src/import-export/csv-imp/gnc-tx-import.hpp

```diff
@@ -73,6 +73,11 @@ extern const gchar* date_format_user[];
  *  - an optional error string
  *  - a struct to hold user selected properties for a transaction
  *  - a struct to hold user selected properties for one or two splits in the above transaction */
+#define PL_INPUT    0
+#define PL_ERROR    1
+#define PL_PRETRANS 2
+#define PL_PRESPLIT 3
+#define PL_SKIP     4
```

Comment:
Yes it can. When writing this part I was considering a scoped enum as that's preferred by Meyers. Such a scoped enum doesn't cast into an int type unless explicitly using static_cast. Got a brain failure at that point and used cpp macros instead of an ordinary enum...

Fixed.